### PR TITLE
First entry for gsoc-jorge

### PR DESCRIPTION
**Pull request by jorgepiloto**
This includes a markdown file about my gsoc experience :rocket:

### CODE REVIEW

File: content/articles/2019-05-10-poliastro-gsoc-jorge.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Title: GSOC-19 starts! 
+Date: 2019-05-10 11:00
+Category: GSCO19
+Tags: GSOC19, poliastro
+slug:_ 2019-05-10-poliastro-gsoc-jorge
```
Comment (astrojuanlu) [inline content/articles/2019-05-10-poliastro-gsoc-jorge.md]: Extra underscore?

File: content/articles/2019-05-10-poliastro-gsoc-jorge.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## From my first pull request to GSOC student
+
+Hi, my name is Jorge and I am one of the selected students for this GSOC
+edition at poliastro under the OpenAstronomy organization. I started working
+on poliastro project after Juanlu came to my
+university to give a talk about Open Source software in engineering. 
+
+I decided to improve API Docs, which gave me an overview on how poliastro was
+build and how the different modules talked among between them. I had no idea
```
Comment (astrojuanlu) [inline content/articles/2019-05-10-poliastro-gsoc-jorge.md]:
```suggestion
built and how the different modules talked among them. I had no idea
```

File: content/articles/2019-05-10-poliastro-gsoc-jorge.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+on how Git and GitHub worked and therefore, when finished with the docs my
+first pull request was wuite massive...
```
Comment (astrojuanlu) [inline content/articles/2019-05-10-poliastro-gsoc-jorge.md]:
```suggestion
first pull request was massive...
```

File: content/articles/2019-05-10-poliastro-gsoc-jorge.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+But after that I started feeling more confortable with GitHub_ and sphinx, the
```
Comment (astrojuanlu) [inline content/articles/2019-05-10-poliastro-gsoc-jorge.md]:
```suggestion
But after that I started feeling more confortable with GitHub and Sphinx, the
```

File: content/articles/2019-05-10-poliastro-gsoc-jorge.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+a propagation algorithm capable of working with all possible orbits, no matter
+their eccentricity.
+
+This would be amazing, since no integrators would be needed for near parabolic
+orbits where numerical methods start failing. The algorithm that we expect to
+solve for this is the one developed by Gauss.
+
+## My plans for this summer
+
+GSOC stippend is one of the main attractives for students when working in Open
```
Comment (astrojuanlu) [inline content/articles/2019-05-10-poliastro-gsoc-jorge.md]:
```suggestion
GSOC stipend is one of the main attractions for students when working in Open
```

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Some typos!

And in "my plans for this summer" I would include the things you will contribute to the project, rather than talking about how you will spend the money :)

**Comment by jorgepiloto** [comment]
I finally was able to remove the `_`. After solving for typos I wrote in a different perspective the last section as suggested :+1:

**Review by astrojuanlu — APPROVED** [review]

**Pull request merged**